src/datastore/pg-store.ts

```diff
@@ -3851,4 +3860,9 @@ export class PgStore {
       return { results: parsed, total: totalQuery[0].count };
     });
   }
+
+  async getConfigState(): Promise<DbConfigState> {
+    const queryResult = await this.sql<DbConfigState[]>`SELECT * FROM config_state`;
+    return queryResult[0];
+  }
```

Comment:
This doesn't seem to be needed in `PgStore`, can you move it back to `PgWriteStore`?